README.md

```diff
@@ -11,8 +11,9 @@
 - [UDS+ Connectathon Page](https://confluence.hl7.org/pages/viewpage.action?pageId=161056877)
 
 ## Version
-**Test Kit:** 1.0.7
-**IG:** 1.0.1
+**Test Kit:** 1.0.8
+**IG:** 1.0.2
 
-### Note From the Developers
-There are known issues with validating certain codesets (not resource structure) when running this kit locally. If you are running locally and encounter an "Unknown Code" error for a given resource, try running that same resource on the hosted version of the kit and see if the codeset can be validated there.
\ No newline at end of file
+### Notes From the Developers
+- There are known issues with validating certain codesets (not resource structure) when running this kit locally. If you are running locally and encounter an "Unknown Code" error for a given resource, try running that same resource on the hosted version of the kit and see if the codeset can be validated there.
+- The validator service takes significant time to assemble once a user begins running the Test Kit locally. As of v1.0.8 of the Inferno Test Kit, please wait up to 45 mins before running any tests to allow this process to complete.
```

Comment:
45 Minutes!?!  We haven't seen it take this long on our end before. We are limited to however long it takes for the validator service to download necessary resources, but it shouldn't take this long... perhaps this was a transient problem.  If this is a consistent problem we can look at ways to mitigate it.

Reply:
I had restarted the service four times today. In every case in which I allowed it to stay open for long enough, it took 35-45 minutes to build. I am not sure if this is an issue with my machine or with the service, but it seemed consistent during my testing.